mods/parsoid.js

```diff
@@ -618,7 +614,7 @@ PSP.makeTransform = function(from, to) {
             var innerRes = res.body[to];
             innerRes.status = 200;
             // Handle bodyOnly flag
-            if (to === 'html' && req.body.bodyOnly) {
+            if (from === 'sections' && to === 'html' && req.body.bodyOnly) {
```

Comment:
Why is that added? In the [docs](https://rest.wikimedia.org/en.wikipedia.org/v1/?doc#!/Transforms/post_transform_html_to_html_title_revision) we state that 'bodyOnly' is valid for other `from` formats as well.